code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -411,6 +411,12 @@
 	TCFLcard["veteran"] = /obj/item/clothing/accessory/badge/tcfl_papers/service/veteran
 	gear_tweaks += new /datum/gear_tweak/path(TCFLcard)
 
+/datum/gear/accessory/syntheticcard
```

Comment:
Given it is for IPC, it should probably be over in `loadout/loadout_xeno/machine.dm` instead and be whitelist restricted.
`whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)`

Reply:
Thanks for the heads up, I was wondering how to whitelist a loadout item.